README.md

```diff
@@ -66,6 +66,14 @@ Thermal simulation is performed under the following conditions:
 
 Newest entries at the top.
 
+
+### Komar V1.1 (February 2021)
```

Comment:
```suggestion
### Komar V1.2 (February 2021)
```

Reply:
I am not sure if it is correct as the device itself was not changed, only the matching accessories.

Reply:
Whether the device is changed is irrelevant because the subject of versioning is not the device but the project. Keeping the old number defeats the purpose of versioning.

https://en.wikipedia.org/wiki/Change_control